blurb/tests/fail/bpo-.rst

```diff
@@ -1 +1 @@
-bpo-12345: Fixed some problem or other.
\ No newline at end of file
+gh-12345: Fixed some problem or other.
```

Comment:
There should be a failing test for `GH-*`.

Reply:
I've added another test for `gh-`